### PR TITLE
Adding RAD Lab UI Overview video

### PR DESCRIPTION
**Pull request by guptamukul-google**

### CODE REVIEW

File: docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md [hunk 1]
```diff
@@ -8,6 +8,18 @@ This is the primary flow and purpose for RAD Lab UI. A step-by-step graphical in
...
 Admins and Users are both able to deploy modules.
 
+Below video provides a quick overview on how to setup RAD Lab UI and deploy RAD Lab modules with the same.
+
+<iframe
+    width="980"
+    height="480"
+    src="https://www.youtube.com/embed/Ir79gCzleLU"
+    frameborder="0"
+    allow="autoplay; encrypted-media"
```
Comment (acolver) [inline docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md:18]: Will this autoplay? I don't think that's desireable.

File: docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md [hunk 2]
```diff
@@ -8,6 +8,18 @@ This is the primary flow and purpose for RAD Lab UI. A step-by-step graphical in
 
 Admins and Users are both able to deploy modules.
 
+Below video provides a quick overview on how to setup RAD Lab UI and deploy RAD Lab modules with the same.
```
Comment (acolver) [inline docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md:11]: The video below provides a quick overview on how to setup RAD Lab UI variables and deploy RAD Lab modules.

**Review by acolver — CHANGES_REQUESTED** [review]

File: docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md (hunk 1 above)
Comment (guptamukul-google) [reply docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md]: Removed autoplay

File: docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md (hunk 2 above)
Comment (guptamukul-google) [reply docs/docs/rad-lab-ui/ui_deployment/deploy-a-module.md]: Updated verbose.

**Review by acolver — APPROVED** [review]

**Pull request merged**